.github/tests/test_model_regression_test_read_dataset_branch_tmpl.py

```diff
@@ -0,0 +1,22 @@
+import pathlib
+import subprocess
+import pytest
+
+TEMPLATE_FPATH = ".github/templates/model_regression_test_read_dataset_branch.tmpl"
+REPO_DIR = pathlib.Path("").absolute()
+TEST_DATA_DIR = str(pathlib.Path(__file__).parent / 'test_data')
+DEFAULT_DATASET_BRANCH = "main"
+
+
+@pytest.mark.parametrize(["comment_body_file", "expected_dataset_branch"],
+                         [
+                            ("comment_body.json", "test_dataset_branch"),
+                            ("comment_body_no_dataset_branch.json", "main")
+                         ])
+def test_read_dataset_branch(comment_body_file, expected_dataset_branch):
+    CMD = ("gomplate "
```

Comment:
I wonder if we should lowercase `CMD` to `cmd` like with other local variables. I probably caused this by making the same mistake in https://github.com/RasaHQ/rasa/pull/10804/files